shared/common-adapters/copyable-text.js.flow

```diff
@@ -0,0 +1,11 @@
+// @flow
+import {Component} from 'react'
+
+export type Props = {
+  value: string,
+  style?: Object,
+  extras$desktop?: HTMLTextAreaElement,
```

Comment:
I've left it so that someone can attach special action listeners to the text field, or so that they can kill the auto selecting nature of the field. I think it may have use in the future, and I've changed the format to match your above comment, so I like that it mirrors the `copyable-text.native` props now.